### ISSUE TITLE
Bug In scrollBugFix Add-on

### ISSUE THREAD

**Issue by metaskills** [FrDH/jQuery.mmenu#532]
I think my issue is described in #332 and I have been using the scrollBugFix add-on. But I found today that the add-on was not working. When I dove in, I found it has [this guard](https://github.com/FrDH/jQuery.mmenu/blob/master/src/addons/scrollbugfix/jquery.mmenu.scrollbugfix.js#L26) in place:

```javascript
if ( !$[ _PLUGIN_ ].support.touch || !this.opts.offCanvas || !this.opts.offCanvas.modal ) {
  return;
}
```

The `!this.opts.offCanvas.modal` was keeping it from executing. However, there is no `modal` option for the [offCanvas add-on](http://mmenu.frebsite.nl/documentation/addons/off-canvas.html). Perhaps this is supposed to be:

```javascript
this.opts.offCanvas.blockUI != 'modal'
``` 

In the short term, I just set `offCanvas: { modal: true }` in my options and have found this solves #332

**Comment by FrDH** [comment]
I thnk you're right.
The odd thing is, the addon does seem to work even though it shouldn't do anything due to this issue. :/

**Comment by FrDH** [comment]
I'll fix this issue in the next version, should be ready shortly.
For the record, it should have been
````javascript
!this.opts.offCanvas.blockUI
````

**Comment by metaskills** [comment]
Thanks. I did notice that:

* blockUI has a default of `true` in [source](https://github.com/FrDH/jQuery.mmenu/blob/master/src/addons/offcanvas/jquery.mmenu.offcanvas.js#L154).
* blockUI has a default of `false` in [documentation](http://mmenu.frebsite.nl/documentation/addons/off-canvas.html#h3).

So I am working on the default of true so your proposed patch makes perfect sense to me too.

**Comment by FrDH** [comment]
the docs will be changed with the next update, thanks for pointing it out.

**Comment by FrDH** [comment]
Should be fixed in version 5.6.2
